gcsio/pom.xml

```diff
@@ -142,27 +160,38 @@
       <groupId>com.google.code.gson</groupId>
       <artifactId>gson</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.cloud.bigdataoss</groupId>
+      <artifactId>util-hadoop</artifactId>
+      <version>${hadoop.identifier}-2.2.3-SNAPSHOT</version>
+      <scope>compile</scope>
+    </dependency>
```

Comment:
Remove this dependency it brings in Hadoop classes.

Reply:
I removed it.